packages/node/src/lib/changes/changes.ts

```diff
@@ -0,0 +1,43 @@
+import { AxiosInstance } from 'axios';
+import { IChanges } from './changes.interface';
+
+export class Changes implements IChanges {
```

Comment:
It must be `export class Changes extends WithHttp implements iChanges {`. This means you don't have to set the private http nor writing the constructor.